packages/web/src/components/containers/ShowcaseContainer.tsx

```diff
@@ -31,7 +31,7 @@ export default function ShowcaseContainer() {
     showcase.showcase?.showcasedScans ?? []
   ).map(({ hostname, webPage, scanPreview }, idx) => ({
     id: idx.toString(),
-    sourceIcon: '',
+    sourceIcon: `/favicons/${hostname.hostname}`,
```

Comment:
A bit too late on my part, but can we move the favicon url generation to a separate fn somewhere?
```
export const getHostnameFaviconUrl = (hostname) => `/favicons/${hostname}`;
```